spec/lens/trace_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'spec_helper'
+
+RSpec.describe Lens::Trace do
+  describe '.process' do
+    let(:first_event) { ['start_processing.action_controller', Time.now, Time.now, 1, {}] }
+    let(:last_event) do
+      [
+        'process_action.action_controller',
+        Time.now,
+        Time.now,
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.